src/main/resources/templates/repo/web/acl/geo-restriction-statement.json

```diff
@@ -0,0 +1,20 @@
+{
+	"OrStatement": {
+        "Statements": [
+            {
+                "GeoMatchStatement": {
+                    "CountryCodes": [ "CN", "HK", "MO", "RU", "IR", "KP", "CU", "VE" ]
+                }
+            },
+			{
+				"GeoMatchStatement": {
+                    "CountryCodes": [ "CN", "HK", "MO", "RU", "IR", "KP", "CU", "VE" ],
+					"ForwardedIPConfig": {
+					    "HeaderName": "X-Forwarded-For",
+						"FallbackBehavior": "MATCH"	
```

Comment:
Note, FallbackBehavior=MATCH ,means match if the header is present but not a valid IP address.  If the header is missing, the rule is not applied.  https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-webacl-forwardedipconfiguration.html